README.md

```diff
@@ -31,35 +24,58 @@ for details on each major and minor release.
 ## Development
 
 1. clone this repo
-2. run `yarn && yarn bootstrap`
+2. run `yarn && yarn build:full`
+
+
+### Modules
 
 
 ### Commands
-This project uses [Lerna](https://github.com/lerna/lerna) to maintain
-multiple npm packages within this repo. As such, all commands should be
-run from the root directory.
-
-Command          | Description
----------------- | ------------------------------------------------------
-`yarn bootstrap`      | Bootstraps dependencies for packages with Lerna
-`yarn build`          | Builds unversioned `lib/` or `dist/` dirs in each package; builds all docs to `<root>/docs/`
+
+Incomplete list of yarn targets.
+
+#### Builds
+
+Some modules rely on others to be built first. It's a good idea to start with a `yarn
+build:full` before running focused builds.
```

Comment:
I'm torn here. I know this is for performance, but I also wonder the safety of just explicitly building those packages that each package may depend on. This is one of those things where if I read it, I'm going to go "hmm... I don't know what this means, or what those packages are... I'm just going to build the full thing... or they just use the command without reading the readme, and figure out later through some Slack conversation that there were instructions to using these commands.